src/main/java/com/kaziamyr/onlinebookstore/OnlineBookStoreApplication.java

```diff
@@ -1,13 +1,37 @@
 package com.kaziamyr.onlinebookstore;
 
+import com.kaziamyr.onlinebookstore.model.Book;
+import com.kaziamyr.onlinebookstore.service.BookService;
+import java.math.BigDecimal;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
 public class OnlineBookStoreApplication {
+    private final BookService bookService;
+
+    public OnlineBookStoreApplication(BookService bookService) {
+        this.bookService = bookService;
+    }
```

Comment:
U should add annotation Autowired for it, but I'm not sure

Reply:
It's no a requirement to mark a constructor with `Autowired`. Spring will understand it anyway. And if I'm not mistaken, you can pass bean directly to commandLineRunner method params.